src/pages/Defi/components/FeaturedCards/FeaturedCard.tsx

```diff
@@ -0,0 +1,97 @@
+import { Box, Button, Flex, Tag, useColorModeValue } from '@chakra-ui/react'
+import { fromAssetId } from '@shapeshiftoss/caip'
+import { PairIcons } from 'features/defi/components/PairIcons/PairIcons'
+import qs from 'qs'
+import { useCallback, useMemo } from 'react'
+import { useHistory, useLocation } from 'react-router'
+import { Amount } from 'components/Amount/Amount'
+import { AssetIcon } from 'components/AssetIcon'
+import { Card } from 'components/Card/Card'
+import { RawText, Text } from 'components/Text'
+import type { StakingEarnOpportunityType } from 'state/slices/opportunitiesSlice/types'
+
+export const FeaturedCard: React.FC<StakingEarnOpportunityType> = ({
+  underlyingAssetIds,
+  assetId,
+  opportunityName,
+  apy,
+  tvl,
+  icons,
+  provider,
+  chainId,
+  contractAddress,
+  rewardAddress,
+  version,
+}) => {
+  const location = useLocation()
+  const history = useHistory()
+  const textShadow = useColorModeValue('0 2px 2px rgba(255,255,255,.5)', '0 2px 2px rgba(0,0,0,.3)')
+  const hoverBgColor = useColorModeValue('gray.100', 'gray.900')
+  const bgIcons = useMemo(() => {
```

Comment:
Don't have to implement this one
```suggestion
  const backgroundIcons = useMemo(() => {
```